templates/model_package_group/sdk_delete_post_request.go.tpl

```diff
@@ -0,0 +1,8 @@
+    if err == nil {
+            if foundResource, err := rm.sdkFind(ctx, r); err != ackerr.NotFound {
+                if isDeleting(foundResource) {
```

Comment:
surakota@ discuss do we need this?

Reply:
Discussion resulted in consensus that is not needed for sagemaker resources